README.md

```
@@ -64,22 +64,40 @@ cargo add zephyr-mcumgr
 
 ```none
-$ zephyr-mcumgr --serial COM42 raw read 0 0 '{"d":"Hello World!"}'
-{
-  "r": "Hello World!"
-}
+$ zephyr-mcumgr --serial COM42
+Device alive and responsive.
+```
+
+Connect to a USB serial port using USB VID/PID:
+```none
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)